docs/framework/data/adonet/updating-data-sources-with-dataadapters.md

```diff
@@ -290,7 +290,7 @@ class Program {
       String primaryCols = String.Join(",", primaryColumns.Select(col => col.ColumnName));  
       String resetCols = String.Join(",", resetColumns.Select(col => "Max(" + col.ColumnName + ") as " + col.ColumnName));  
```

Comment:
not the `string.Format` call, but still this is also the candidate for string interpolation?
```csharp
col => $"Max({col.ColumnName}) as {col.ColumnName}"
```

Reply:
I've replaced the string concatenation with string interpolation.

Reply:
Thanks!